server/ig-main.js

```diff
@@ -0,0 +1,63 @@
+const Client = require('instagram-private-api').V1;
+const device = new Client.Device('eatifyjohn');
+const path = require('path');
+const cookiePath = path.join(__dirname, '/cookies/eatifyjohn.json');
+const storage = new Client.CookieFileStorage(cookiePath);
+
+function IG() {
+
+}
+
+IG.prototype.getAccountByName = function(username, session) {
```

Comment:
don't think there is a need for the `IG` function, and adding things to the prototype. I think methods like `getAccountByName`, `getAccountById`, can all just be functions that take in the required arguments along with a session.